src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
I'm not sure how to type the return type. From what I saw it can be: `RouterConfiguration`, any, void, and Promise and PromiseLike that resolve to these.